sklearn/neighbors/_kde.py

```diff
@@ -69,6 +69,25 @@ class KernelDensity(BaseEstimator):
         Additional parameters to be passed to the tree for use with the
         metric.  For more information, see the documentation of
         :class:`BallTree` or :class:`KDTree`.
+
+    See Also
+    --------
+    sklearn.neighbors.KDTree : K-dimensional tree for fast generalized N-point
+        problems.
+    sklearn.neighbors.BallTree : Ball tree for fast generalized N-point
+        problems.
+
+    Examples
+    --------
+    Compute a gaussian kernel density estimate with a fixed bandwidth.
+    >>> import numpy as np
+    >>> rng = np.random.RandomState(42)
+    >>> X = rng.random_sample((100, 3))
+    >>> kde = KernelDensity(kernel='gaussian', bandwidth=0.5).fit(X)
+    KernelDensity(...)
```

Comment:
This should have been removed as well, as it make doctest fail. Fixed in https://github.com/scikit-learn/scikit-learn/commit/726eac9c295e804d0a5587af25b9fc9d5856a1fe on master.

Reply:
oops, thanks !